core/node/api_server/src/web3/tests/vm.rs

```diff
@@ -557,12 +567,19 @@ impl HttpTest for TraceCallTestAfterSnapshotRecovery {
     ) -> anyhow::Result<()> {
         self.fee_input.expect_default(TraceCallTest::FEE_SCALE);
         let call_request = CallTest::call_request(b"pending");
-        let call_result = client.trace_call(call_request.clone(), None, None).await?;
+        let CallTracerResult::CallTrace(call_result) =
```

Comment:
Nit: You can introduce conversion methods for `CallTracerResult` like `unwrap_hierarchical` (`unwrap_default`?) / `unwrap_flat` to make them shorter.